ResourceStatusSystem/Utilities/CSHelpers.py

```diff
@@ -215,15 +215,18 @@ def getStorageElementEndpoint( seName ):
     gLogger.warn( "Could not get SE parameters", "SE: %s" % seName )
     return seParameters
 
-  host = seParameters['Value']['Host']
-  port = seParameters['Value']['Port']
-  wsurl = seParameters['Value']['WSUrl']
-
-  # MAYBE wusrl is not defined
-  if host and port:
-    url = 'httpg://%s:%s%s' % ( host, port, wsurl )
-    url = url.replace( '?SFN=', '' )
-    return S_OK( url )
+  if seParameters['Value']['PluginName'] == 'GFAL2_SRM2':
```

Comment:
Rather than testing on the PluginName, I would test on the protocol. There is a `Protocol` field in the dictionary which is equal to `srm`